ci/scripts/collect-test-robot.sh

```diff
@@ -38,20 +38,7 @@ yum install python3-devel -y
 # Get OS version id
 VERSION_ID=$(grep '^VERSION_ID' /etc/os-release | sed -En 's/^VERSION_ID="([[:digit:]])\.[[:digit:]]"/\1/p')
 
-# Force version for el7 only
-if [ -f /etc/os-release ]; then
-    case "$VERSION_ID" in
-        7)
-            pip3 install grpcio==1.33.2 grpcio_tools==1.33.2
-            ;;
-        8)
-            pip3 install grpcio grpcio_tools
-            ;;
-        *)
-            echo "OS Version is neither 7 or 8"
-            ;;
-    esac
-fi
+pip3 install grpcio==1.33.2 grpcio_tools==1.33.2
```

Comment:
bad choice. This was the one for el7.

`pip3 install grpcio grpcio_tools`
works on alma8 and alma9